### PR TITLE
fix(apis/web/resource/view): 修复资源绑定插件未修改但触发更新问题

### PR DESCRIPTION
**Pull request by F-cq**
### Description

修复资源绑定插件未修改但触发更新问题

Fixes # (issue)

### Checklist

- [ ] 填写 PR 描述及相关 issue (write PR description and related issue)
- [ ] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [ ] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

File: src/dashboard/apigateway/apigateway/apis/web/plugin/views.py [hunk 1]
```diff
@@ -327,6 +328,25 @@ class PluginConfigRetrieveUpdateDestroyApi(
...
+        try:
+            input_yaml = yaml_loads(input_data["yaml"])
+            current_yaml = yaml_loads(instance.yaml)
+
+            if input_yaml != current_yaml:
+                return True
+        except Exception:
+            return True
+
+        if input_data["name"] != instance.name:
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apis/web/plugin/views.py:342]: name 不需要 check

Comment (wklken) [reply src/dashboard/apigateway/apigateway/apis/web/plugin/views.py:342]: 对应测试用例一并去掉

File: src/dashboard/apigateway/apigateway/apis/web/plugin/views.py [hunk 2]
```diff
@@ -327,6 +328,25 @@ class PluginConfigRetrieveUpdateDestroyApi(
...
     lookup_field = "id"
 
+    def _check_if_changed(self, input_data: Dict[str, Any], instance: PluginConfig) -> bool:
+
+        try:
+            input_yaml = yaml_loads(input_data["yaml"])
+            current_yaml = yaml_loads(instance.yaml)
+
+            if input_yaml != current_yaml:
+                return True
```
Comment (Han-Ya-Jun) [inline src/dashboard/apigateway/apigateway/apis/web/plugin/views.py:338]: 这里直接 return  input_yaml == current_yaml ?

File: src/dashboard/apigateway/apigateway/apis/web/plugin/views.py (hunk 1 above)
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/apis/web/plugin/views.py:342]: +1

Comment (F-cq) [reply src/dashboard/apigateway/apigateway/apis/web/plugin/views.py]: 已修复

File: src/dashboard/apigateway/apigateway/apis/web/plugin/views.py (hunk 2 above)
Comment (F-cq) [reply src/dashboard/apigateway/apigateway/apis/web/plugin/views.py]: 已修复

**Review by wklken — APPROVED** [review]

**Pull request merged**